.github/workflows/build-toolchain.yml

```diff
@@ -136,7 +136,7 @@ jobs:
 
             if ("${{ inputs.preset }}" -like "win32*") {
             # For win32 preset, look in config-specific subdirectories
-            $configToUse = if ("${{ inputs.preset }}" -eq "win32dbg") { "Debug" } else { "Release" }
+            $configToUse = if ("${{ inputs.preset }}" -eq "win32-debug") { "Debug" } else { "Release" }
```

Comment:
This looks odd. What should be done with it? We have more than one debug configuration.

Reply:
Chat gippity helped me fixing this thing, so this builds now.